response_operations_ui/templates/collection_exercise/ce-collection-instrument-eq-section-redacted.html

```diff
@@ -26,8 +26,40 @@
             <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
             <input type="hidden" name="csrf_token" value="" />
             <input type="hidden" name="ce_id" value="{{ ce.id }}" />
-            <div class="ons-grid ons-col-12@m ons-grid-flex--vertical-center ons-grid--no-wrap@m ons-u-mt-m ons-u-mb-m">
-                {% for eq_ci in all_cis_for_survey|batch(3, ' ') %}
+           <div class="ons-grid ons-col-12@m ons-grid-flex--vertical-center ons-grid--no-wrap@m ons-u-mt-m ons-u-mb-m">
```

Comment:
Have left the existing flow the same outside of CIR, but refactored the table rendering outside of the for loops and removed some redundant batching for when we more to the CIR flow.